calico-cloud_versioned_docs/version-3.17/image-assurance/install-the-admission-controller.mdx

```diff
@@ -61,36 +61,44 @@ Container admission policies are custom Kubernetes resources that allow you to c
 
    :::caution
 
-   If you generate the key and certificate pair yourself, you must set the SANS to `tigera-image-assurance-admission-controller-service.tigera-image-assurance.svc`
+   If you generate the key and certificate pair yourself, you must set the SANS to `tigera-image-assurance-admission-controller-service.tigera-image-assurance.svc`.
 
    :::
 
 1. Download and configure the Admission Controller manifests.
 
-   As a safety mechanism, we require that you specify the namespaces that the Admission Controller will apply in
-   the `IN_NAMESPACE_SELECTOR_KEY` and the `IN_NAMESPACE_SELECTOR_VALUES`. These values configure the Kubernetes API server to send admission requests to our Admission Controller only for resources from relevant namespaces.
+As a safety mechanism, we require that you specify the namespaces that the Admission Controller will apply in
+the `IN_NAMESPACE_SELECTOR_KEY` and the `IN_NAMESPACE_SELECTOR_VALUES`. These values configure the Kubernetes API server to send admission requests to our Admission Controller only for resources from relevant namespaces.
 
-   For example, to configure the Kubernetes API server to send admission requests for resources created in any namespace with label key `foo`, and label values either `bar` or `baz`, the variables would be set as follows:
+For example, to configure the Kubernetes API server to send admission requests for
+resources created in any namespace with label key `name`, and label values either `prod` or `critical`, set the variables as follows:
 
    ```bash
-   export IN_NAMESPACE_SELECTOR_KEY="foo" && IN_NAMESPACE_SELECTOR_VALUES="bar baz"
+   export IN_NAMESPACE_SELECTOR_KEY="name" && IN_NAMESPACE_SELECTOR_VALUES="prod critical"
```

Comment:
This is a bit of a paranoid view, but is the example of using selector label "critical" here a good idea, given that later on we say that we critical namespaces like `kube-system` should be avoided. 

I recognize it's actually talking about different things, but just wondering if we can use a different term other than critical for one of the two cases to further differentiate them.

Reply:
I'm fine with differentiating. @WilliamTigera What do you suggest I change the "prod critical" to?

Reply:
I'm good with putting it just as prod.

Reply:
Updated all to "staging".